src/components/v5/common/ColonyDashboardHeader/partials/ColonyLinks/ColonyLinks.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC } from 'react';
+import { ShareNetwork } from 'phosphor-react';
+import { useLocation } from 'react-router-dom';
+import { useCopyToClipboard } from '~hooks/useCopyToClipboard';
+import { COLONY_LINK_CONFIG } from '~constants/colonyLinks';
+import { formatText } from '~utils/intl';
+import Tooltip from '~shared/Extensions/Tooltip';
+import Link from '~v5/shared/Link';
+import Menu from '../Menu';
+import { useExternalLinks } from '../Menu/hooks';
```

Comment:
We could add an export to the `index` file of `Menu` and that way we avoid having to write 2 imports for code that is on the same folder: `import Menu, { useExternalLinks } from '../Menu';`. Cleaner this way 👌